src/System.IO.Pipelines/src/System/IO/Pipelines/PipeReader.cs

```diff
@@ -53,10 +53,20 @@ public abstract partial class PipeReader
         /// <summary>
         /// Returns a <see cref="Stream"/> that wraps the <see cref="PipeReader"/>.
         /// </summary>
+        /// <param name="leaveOpen">Optional flag indicating disposing returned <see cref="Stream"/> won't complete <see cref="PipeReader"/>.</param>
```

Comment:
Let's just ~~steal~~ borrow the content straight from `StreamReader`'s equivalent parameter:

```suggestion
        /// <param name="leaveOpen"><see langword="true" /> to leave the pipe open after the <see cref="System.IO.Stream" /> object is disposed; otherwise, <see langword="false" />.</param>
```

Reply:
In the case of StreamReader there's no caching, so there' no "most truthy" behavior which makes the leaveOpen parameter much easier to document accurately.

If we're going to do something implicit an non-obvious like having a true leaveOpen argument affect all previous undisposed streams returned by AsStream() as well as all future streams returned AsStream(), we should at least document that.